# diagnostic_params.yaml
#
# Copyright (C) 2015 Alessandro Francescon
# 
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, either version 3 of the License.
# 
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
# 
# You should have received a copy of the GNU General Public License
# along with this program.  If not, see <http://www.gnu.org/licenses/>.

# The warning value for SAMx8 average load, in percent (default: 10.0)
samx8_load_warning_threshold: 50

# The critical value for SAMx8 average load, in millis (default: 50.0)
samx8_load_critical_threshold: 50.0

# The warning value for SAMx8 average delay, in millis (default: 50.0)
samx8_delay_warning_threshold: 50.0

# The critical value for SAMx8 average delay, in millis (default: 100.0)
samx8_delay_critical_threshold: 100.0
